src/LinearMaps.jl

```diff
@@ -3,14 +3,13 @@ module LinearMaps
 
 export LinearMap, AbstractLinearMap
 
-import Base: +, -, *, \, /, ==, transpose
-
-if VERSION >= v"0.7.0-DEV.1415"
-    const adjoint = Base.adjoint
-else
-    const adjoint = Base.ctranspose
-end
+import LinearAlgebra, SparseArrays
 
+import Base: +, -, *, \, /, ==
+import LinearAlgebra: transpose, adjoint
+import LinearAlgebra: issymmetric, ishermitian, isposdef
```

Comment:
Personally, I am more a fan of using fully qualified names when overloading methods (see below) and thus, not importing all of them. I did seem to make an exception for operators in the old version, but even there `Base.:(+)` is not too bad.

Reply:
I thought one needs to import those explicitly if one wants to overload them. I didn't mean to omit full qualifiers, that was more like ignorance. I'll certainly add the full qualifiers in the code and check if it is necessary to import overloaded functions here.

Reply:
If I understood correctly, and nothing has changed.
* If you only want to use a function (or anything else) from a package (including standard library packages), you do
    *   `using Package` if the function/object/name is exported
    *  `using Package` followed by `using Package: unexported_name` if the name is not exported. Or just `using Package` and then use the fully qualified name `Package.unexported_name`.

* If you want to overload a function (exported or not), you should:
    *   `using Package` followed by `import Package: name`
    *  or `using Package` and then just use the fully qualified name `Package.name`.

Reply:
You might want `using LinearAlgebra`, unless you also want to use fully qualified names whenever you just want to use/call one of these functions.